52/PedroDevOps/pomodoro.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+"""
+Author:  Pedro DevOps <pedroDevOps@gmail.com>
+Purpose: Create my own Pomodoro Timer
+Project: 100Days of code with Python
+Progress: Round1, Day3 = R1D3
+Challenge: https://codechalleng.es/challenges/52/
+OBS:
+    flake8 was used: # flake8 pomodoro.py
+    autopep8 was used: # autopep8 --in-place --aggressive --aggressive pomodoro.py
+    bandit was used: # bandit -r pomodoro.py
+"""
+
+from datetime import timedelta, datetime
+from time import sleep
+from os import system
+import argparse
+
+
+# Fuction that will do the countdown and print on terminal
+def countdown(title, time_to_focus, time_to_rest):
+    system('clear')
+    stop_time = datetime.now() + time_to_focus
+    resume_time = stop_time + time_to_rest
+
+    # Loop for the focus time
+    while datetime.now() < stop_time:
+        delta = stop_time - datetime.now()
+        totalMinute, second = divmod(delta.seconds, 60)
+        hour, minute = divmod(totalMinute, 60)
+        print(title, f"{hour}:{minute:02}:{second:02}", end='\r')
+        sleep(1)
+
+    print("Well done! Let's rest for: {}".format(time_to_rest))
+
+    # Loop for the break time
+    while datetime.now() < resume_time:
+        delta = resume_time - datetime.now()
+        totalMinute, second = divmod(delta.seconds, 60)
+        hour, minute = divmod(totalMinute, 60)
+        print(title, f"{hour}:{minute:02}:{second:02}", end='\r')
+        sleep(1)
+    system('clear')
+
+
+def pomodoro_timer(title, interval_time, rest_time, how_many_cycles):
+    keep_using_timer = True
+    while keep_using_timer:
+        try:
+            i = 0
+            while i < how_many_cycles:
```

Comment:
if the amount of known you can use a `for` loop, e.g. `for i in range(how_many_cycles)`